.github/workflows/conformance.yml

```diff
@@ -234,7 +234,7 @@ jobs:
         mongodb-replica-set: test-rs
 
     - name: Set up Go
-      uses: actions/setup-go@v3
+      uses: actions/setup-go@v4
```

Comment:
When updating to setup-go@v4, please disable the cache, or it will be cached twice

Reply:
Just set `cache: false` in `with`
We have our own cache down below